From 49491bf5da7327925bbdafd94a2b04ddd99797f2 Mon Sep 17 00:00:00 2001
From: Amelia Lu <peirulu@amazonaws.com>
Date: Wed, 4 Feb 2026 22:29:58 +0000
Subject: [PATCH] replace logstash formatter to default

The logrus-logstash-hook does not have LogstashFormatter in version v1.0.0,
replace it with the default formatter.s
---
 registry/registry.go | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/registry/registry.go b/registry/registry.go
index 4b81e4d9..a946ba17 100644
--- a/registry/registry.go
+++ b/registry/registry.go
@@ -359,9 +359,7 @@ func configureLogging(ctx context.Context, config *configuration.Configuration)
 			TimestampFormat: time.RFC3339Nano,
 		})
 	case "logstash":
-		log.SetFormatter(&logstash.LogstashFormatter{
-			TimestampFormat: time.RFC3339Nano,
-		})
+		log.SetFormatter(logstash.DefaultFormatter(log.Fields{}))
 	default:
 		// just let the library use default on empty string.
 		if config.Log.Formatter != "" {
-- 
2.34.1

